wazo_call_logd/plugins/cdr/api.yml

```diff
@@ -437,6 +437,18 @@ definitions:
         - inbound
         - internal
         - outbound
+      remote_party:
```

Comment:
A description of this field would be welcome.

Reply:
Done